modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -281,9 +281,9 @@ will only return label `:A`, because that is the only label for which traversal
 [[access-control-limitations-db-operations]]
 == Security and count store operations
 
-The rules of a security model may impact some of the database operations.
-This means extra security checks are necessary to incur additional data accesses, especially in the case of count store operations.
-These are, however, usually very fast lookups and the difference might be noticeable.
+The rules of a security model may impact some database operations.
+This is because extra security checks are necessary, especially in the case of count store operations.
+These are, however, usually very fast lookups and the difference might not be noticeable.
```

Comment:
I think this section actually was supposed to point out that count operations that without privileges might be fast can get slow due to extra checks needed (and not being able to use count store for example)?

Reply:
For example look at the table just above your new property rule section

Reply:
hmmm, I just think this sentence is strange. I added the "not" because I thought we tried to express that even if it got slower it might not be noticeable since it is fast. But if it the other way around I can remove it, or maybe we can express it more clearly somehow.

Reply:
I'm pretty sure it will get noticeable slower since we no longer can use the fast version and that that is what it was supposed to point out.

You also removed the part about extra additional data accesses which I think is still accurate/relevant?